src/fileWriters/createTemplateFiles.js

```diff
@@ -0,0 +1,37 @@
+import path from 'path';
+import fse from 'fs-extra';
+import chalk from 'chalk';
+import mv from 'mv';
+
+import { SOURCE_FOLDER, TEMPLATES_DIR } from '../constants';
+import templatizeFile from './templatizeFile';
+
+const createTemplateFiles = async () => {
+  try {
+    const templateFileNames = {
+      'package.json': 'package.template.json',
+      'docker-compose.yml': 'docker-compose.template.yml',
+    };
+
+    Object.keys(templateFileNames).forEach((fileName) => {
```

Comment:
Correct me if I'm wrong, but this would take `package.json`, rename it to `package.template.json` and then overwrite `package.template.json` with some custom data? Why not just keep the file contents of `package.json` in-memory, overwrite it with the custom data and skip the rename step?

Reply:
Haha yeah, I realized the whole rename game is no longer needed now and can just overwrite the data directly in `package.json` and `docker-compose.yml`. I will refactor accordingly.